weatherAplication/Modules/Home/View/HomeViewController.swift

```diff
@@ -50,6 +57,12 @@ class HomeViewController: UIViewController {
         aplicationNameLabel.backgroundColor = .black
         view.addSubview(aplicationNameLabel)
 
+        lastCityNamesTable.dataSource = self
+        lastCityNamesTable.delegate = self
+        lastCityNamesTable.backgroundColor = .cyan
+        view.addSubview(lastCityNamesTable)
+        lastCityNamesTable.register(UITableViewCell.self, forCellReuseIdentifier: "cell")
```

Comment:
tableView, nie samo table. Poza tym jak setupujesz dużo properties danego obiektu to zrób na to osobną metodę